consensus/receiver.py

```diff
@@ -16,6 +16,8 @@
 if config.INFURA_URL.count('rinkeby') > 0 or config.INFURA_URL.count('idchain') > 0:
     w3.middleware_onion.inject(geth_poa_middleware, layer=0)
 
+NUM_SEALERS = 10
```

Comment:
initialization can be ignored as we are retrying on errors while updating this number.